src/resources/snippets.ts

```diff
@@ -29,6 +29,12 @@ export default [
     "code": ".main-likedSongsButton-likedSongsIcon {\n  color: var(--spice-sidebar);\n  background: var(--spice-text);\n}",
     "preview": "resources/assets/snippets/fix-liked-icon.png",
   },
+  {
+    "title": "Fix 'DJ' Icon",
+    "description": "Makes the DJ icon match themes better",
+    "code": ".GGx57b6ZwGgzUeTpynUw { background: var(--spice-text); color: var(--spice-sidebar); }",
```

Comment:
```suggestion
    "code": ".main-collectionLinkButton-icon > div { background: var(--spice-text); color: var(--spice-sidebar); }",
```
Use relative selectors of mapped classes instead of unmapped classes.